src/docs/product/data-management-settings/server-side-sampling/getting-started.mdx

```diff
@@ -40,19 +40,25 @@ For more detailed information about each of these, check out our [full Current L
 
 Once you’ve confirmed that your application only relies on [supported SDKs](#current-limitations), then the next step is to update them to the specified version or higher. Check out the migration documentation for the [SDKs](/platforms/) you’re using to learn everything you need to know to get up and running again with the latest Sentry features.
 
-## 2. Set a Uniform Sampling Rate
+## 2. Set a Global Sample Rate
```

Comment:
I would stick with "uniform" sampling rate. "Global" makes me think of something bigger, like org-wide or something like that.

Reply:
I think it's fine to stick with _uniform_ elsewhere, but for the heading, we should match the UI text.